types/testingutils/ssv_msgs.go

```diff
@@ -398,7 +406,7 @@ var TestSelectionProofWithJustificationsConsensusData = func(ks *TestKeySet) *ty
 
 	return &types.ConsensusData{
 		Duty:                       TestingAggregatorDuty,
-		Version:                    spec.DataVersionBellatrix,
+		Version:                    spec.DataVersionDeneb,
```

Comment:
IMO the version should be reverted to `DataVersionBellatrix`
The duty not related to proposal here

Reply:
it caused failures on ssv due to a bug that @moshe-blox just fixed
anyhow we shouldn't have Bellatrix in our code anymore so nothing wrong with this